Source/WebCore/loader/cache/CachedResourceLoader.cpp

```diff
@@ -990,7 +990,7 @@ ResourceErrorOr<CachedResourceHandle<CachedResource>> CachedResourceLoader::requ
     }
 
     if (Document* document = this->document()) {
-        request.upgradeInsecureRequestIfNeeded(*document);
+        request.upgradeInsecureRequestIfNeeded(*document, ContentSecurityPolicy::ForceUpgradeRequest::No);
```

Comment:
Either we use the default value (we then keep writing upgrade...(*document) or we remove the default value from CachedResourceRequest:: upgradeInsecureRequestIfNeeded